.github/workflows/publish.yml

```diff
@@ -42,12 +42,12 @@ jobs:
 
       - uses: actions/setup-python@v5
         with:
-          python-version: "3.11"
+          python-version: "3.12"
 
       - uses: actions/setup-node@v4
         # node required to build wheel
         with:
-          node-version: "16"
+          node-version: "22"
```

Comment:
I've made this "lts/*" in various other projects to reduce the amount of manual bumping needs - fine to bump manually also